### PR TITLE
[New] mitigate memcached amplification attacks

### PR DESCRIPTION
**Pull request by cwlinode**

### CODE REVIEW

File: docs/security/mitigations/mitigating-memcached-amplification-attacks.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+---
+author:
+  name: Linode
+  email: docs@linode.com
+description: 'Outline of the memcached amplification attack, how it affects you, and how to avoid it.'
```
Comment (GuessWhoSamFoo) [inline docs/security/mitigations/mitigating-memcached-amplification-attacks.md]: Needs a longer description

File: docs/security/mitigations/mitigating-memcached-amplification-attacks.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
+---
```
Comment (GuessWhoSamFoo) [inline docs/security/mitigations/mitigating-memcached-amplification-attacks.md]: Mitigations is missing `_index.md` and will not be visible from the navigation pages

File: docs/security/mitigations/mitigating-memcached-amplification-attacks.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+## I'm running memcached. What do I do?
+
+Memcached should not be exposed to the internet. If your Linode is detected to be running a vulnerable instance of memcached, you will be alerted by our Support team. You can also take steps to secure your memcached instance:
+
+* If you require memcached to communicate over a private network:
+
+    1.  Bind memcached to a specific IP address and port.
+
+    2.  Configure you Linode's firewall. If you need help configuring a firewall on your distribution, see [our firewall guides](/docs/security/firewalls/) for more information.
```
Comment (GuessWhoSamFoo) [inline docs/security/mitigations/mitigating-memcached-amplification-attacks.md]: your

File: docs/security/mitigations/mitigating-memcached-amplification-attacks.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+        sudo systemctl restart memcached
+
+    If your system uses SysV:
+
+        sudo service memcached restart
+
+## I'm not running memcached. Does this affect me?
+
+Potentially, yes. We're experiencing a high load on our network due to this exploit. You may experience dropped packets and a temporary decrease in network performance, regardless of which data center your Linode resides in.
```
Comment (GuessWhoSamFoo) [inline docs/security/mitigations/mitigating-memcached-amplification-attacks.md]: awkward phrasing

File: docs/security/mitigations/mitigating-memcached-amplification-attacks.md (hunk 2 above)
Comment (EdwardAngert) [reply docs/security/mitigations/mitigating-memcached-amplification-attacks.md]: ✔️

**Pull request merged**